routes/rider.js

```diff
@@ -1,10 +1,11 @@
 var express = require('express')
 var router = express.Router()
 var db = require('../model/db.js')
+const connect = require('connect-ensure-login')
 
 /* GET user-rider's rides/bids listing page. */
-router.get('/', async function(req, res, next) {
-  const riderId = req.query.user_id_field
+router.get('/', connect.ensureLoggedIn('/login') ,async function(req, res, next) {
+  const riderId = req.user
```

Comment:
Is there a way to set this as `userId` for passport? Because I'm getting confused between username and userid (especially with no types 😆)

Reply:
Did some quick google-fu, not to my knowledge sorry. It's not as simple as writing `req.userId = req.user` ; `req.user = undefined`; i think due to the fact that `req.user` is required as the default for some methods in passportjs. 

tl;dr seems like there should be a way but i'm not sure